packages/rn-tester/js/examples/FlatList/FlatList-basic.js

```diff
@@ -185,11 +185,12 @@ class FlatListExample extends React.PureComponent<Props, State> {
                 this._setBooleanValue('useFlatListItemComponent'),
               )}
               {/* [TODO(macOS GH#774)  */}
-              {renderSmallSwitchOption(
-                'Keyboard Navigation',
-                this.state.enableSelectionOnKeyPress,
-                this._setBooleanValue('enableSelectionOnKeyPress'),
-              )}
+              {Platform.OS === 'macos' &&
```

Comment:
how would iPad keyboard accessibly work ? :)

Reply:
keyboard events aren't implemented on iOS at all. So this "feature" isn't really available for iPad

Reply:
Wait, huh? I have an iPad that does seem to have some keyboard accessibility features.